seqr/management/commands/reload_saved_variant_json_tests.py

```diff
@@ -0,0 +1,92 @@
+#-*- coding: utf-8 -*-
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+from django.db.models.query_utils import Q
+from seqr.models import Project
+
+PROJECT_NAME = u'1kg project n\u00e5me with uni\u00e7\u00f8de'
+PROJECT_GUID = 'R0001_1kg'
+PROJECT_GUID2 = 'R0003_test'
+SAVED_VARIANT_GUIDS = ['SV0000001_2103343353_r0390_100', 'SV0000002_1248367227_r0390_100']
+FAMILY_ID = '1'
+
+class ReloadSavedVariantJsonTest(TestCase):
+    fixtures = ['users', '1kg_project']
+
+    @mock.patch('logging.getLogger')
+    @mock.patch('seqr.views.utils.variant_utils.update_project_saved_variant_json')
+    def test_with_param_command(self, mock_update_json, mock_get_logger):
+        mock_update_json.return_value = SAVED_VARIANT_GUIDS
+        mock_logger = mock_get_logger.return_value
+
+        # Test with a specific project and a family id.
+        call_command('reload_saved_variant_json',
+                     PROJECT_NAME,
+                     '--family-id={}'.format(FAMILY_ID))
+
+        projects_to_process = [PROJECT_NAME]
+        projects = Project.objects.filter(Q(name__in=projects_to_process) | Q(guid__in = projects_to_process))
```

Comment:
Given that this is querying fixture data, please write a test query that directly gets what you expect (i.e. filtering for the single expected guid)

Reply:
Changed to query for a single project.